lib/android/app/src/main/java/com/wix/reactnativenotifications/fcm/FcmToken.java

```diff
@@ -72,7 +72,21 @@ public void onAppReady() {
     }
 
     protected void refreshToken() {
-        FirebaseMessaging.getInstance().getToken()
+        FirebaseMessaging instance;
+
+        try {
+            instance = FirebaseMessaging.getInstance();
+        } catch (IllegalStateException e) {
+            // This would be expected when only local notifications are used and Firebase is not configured
+            if (BuildConfig.DEBUG) Log.w(
+                LOGTAG,
+                "FirebaseApp is not initialized. See https://wix.github.io/react-native-notifications/docs/installation-android#receiving-push-notifications",
```

Comment:
Feel free to shorten this log or modify as you see fit!

I thought it might be convenient to give an actionable step for developers to take. As it's a debug log, the extra log length couldn't hurt.